components/rhs_comment.jsx

```diff
@@ -39,7 +40,8 @@ export default class RhsComment extends React.Component {
         removePost: PropTypes.func.isRequired,
         previewCollapsed: PropTypes.string.isRequired,
         previewEnabled: PropTypes.bool.isRequired,
-        isEmbedVisible: PropTypes.bool
+        isEmbedVisible: PropTypes.bool,
+        pluginPostTypes: PropTypes.object
```

Comment:
This is used by the code that came from [post_body.jsx](https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/post_body/post_body.jsx#L221). I couldn't find any cases where the prop is ever set when that component is used, so adding it as an optional prop to this component won't hurt anything. Not sure why we expose an unused prop though.

Reply:
It's just used to detect if this post type is being overridden by a plugin. Plugins don't support replies being overridden yet so you can just remove it plus the `hasPlugin` check below. I'll re-add when plugins get support for overriding replies